pkg/network/tracer/tracer_unsupported.go

```diff
@@ -38,3 +38,8 @@ func (t *Tracer) DebugNetworkState(clientID string) (map[string]interface{}, err
 func (t *Tracer) DebugNetworkMaps() (*network.Connections, error) {
 	return nil, ebpf.ErrNotImplemented
 }
+
+// DebugEBPFMaps is not implemented on this OS for Tracer
+func (t *Tracer) DebugEBPFMaps() (string, error) {
```

Comment:
I think you will need one of these in `tracer_windows.go` too.

Reply:
done